File: Cap 05/5.4 - Laco Condicional Pos-Teste/5.4.2 - Controle Condicional Verdadeiro/Efetivo/Iterativo/executeIT.c
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 5 - Topico 5.4.2
 * 
 * Programa ..: LACO CONDICIONAL POS-TESTE COM FLUXO VERDADEIRO ITERATIVO
 */

#include <stdio.h>

int I, N, R;
int main(void) {
  printf("LACO POS-TESTE COM FLUXO VERDADEIRO\n");
  printf("Multiplicacao de numero por 3 por 5 vezes\n\n");
  I = 1;
  do { // como "execute"
    printf("Calculo: %d\n", I);
    printf("Entre valor numerico inteiro: ");
    scanf("%d", &N);
    R = N * 3;
    printf("O numero informado X 3 = %d\n\n", R);
    I = I + 1;
  } while (I <= 5); // como "enquanto_for (I <= 5)"
  return 0;
}

File: Cap 05/5.4 - Laco Condicional Pos-Teste/5.4.2 - Controle Condicional Verdadeiro/Efetivo/Iterativo/executeIT.rb
=begin
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 * 
 * Capitulo ..: 5 - Topico 5.4.2
 * 
 * Programa ..: LACO CONDICIONAL POS-TESTE COM FLUXO VERDADEIRO ITERATIVO
=end

puts 'LACO POS-TESTE COM FLUXO VERDADEIRO'
puts 'Multiplicacao de numero por 3 por 5 vezes'
puts
i = 1
begin # simulacao "execute"
  puts "Calculo: #{i}"
  print 'Entre valor numerico inteiro: '
  n = gets.chomp.to_i
  r = n * 3
  puts "O numero informado X 3 = #{r}"
  puts
  i = i + 1
end while (i <= 5) # como "enquanto_for (I <= 5)"
